src/main/webapp/app/common/secondary/UserDTO.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from '@/common/domain/User';
+
+export interface UserDTO {
```

Comment:
as already said in a previous comment, DTO = Data Transfert Object, we should not use it in Front